aleemer/Internify
# Profile Job Postings List fix

#### User Story
No user story

#### Changes made
- The profile page is supposed to trigger the job posting table if there are no jobs to tell the users to add a job if there's no job postings with that user.

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Log in to a user with no jobs
2. Go to the profile page and you should see this on the right column:
 
<img width="1064" alt="image" src="https://user-images.githubusercontent.com/37882255/128944257-61aed233-472b-4001-8eb0-acbecefb0b6d.png">
